pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
This is what actually happens when the ISR does not satisfy the hardware requesting a level interrupt.
![effects](https://cloud.githubusercontent.com/assets/3876044/13606874/b6396572-e51b-11e5-8952-3d29fae20308.png)